overrides.yaml

```diff
@@ -0,0 +1,189 @@
+additions: []
+removals: []
+changes:
+  - id: prairiewarehouse
+    centerpoint: [-93.146459, 44.454619]
+  - id: seccombe
+    centerpoint: [-93.152084, 44.458878]
+  - id: baldspot
+    centerpoint: [-93.154627, 44.461183]
+  - id: baseballfield
+    centerpoint: [-93.147161, 44.466607]
+  - id: bellfield
+    centerpoint: [-93.148501, 44.460168]
+  - id: 3oaks
+    centerpoint: [-93.147258, 44.463252]
+  - id: hillsideprairie
+    centerpoint: [-93.147727, 44.471211]
+  - id: kettleholemarsh
+    centerpoint: [-93.138936, 44.475456]
+  - id: littlebaldspot
+    centerpoint: [-93.151278, 44.460557]
+  - id: lymanlakes
+    centerpoint: [-93.151267, 44.462789]
+  - id: maifete
+    centerpoint: [-93.151733, 44.463059]
+    name: Mai-Fête Island
```

Comment:
This is the only name key of all the overrides?

Reply:
It's the only one I wanted to override. The override lookup is based on id, not name.

Reply:
(The original name doesn't have the accent, but Google does, so I wanted to include it.)